libhpcalcs/src/prime_vpkt.c

```diff
@@ -83,6 +83,92 @@ HPEXPORT void HPCALL prime_vtl_pkt_del(prime_vtl_pkt * pkt) {
     }
 }
 
+HPEXPORT int HPCALL prime_send_new_protocol_init(calc_handle * handle) {
+    int res;
+
+    // This is a special package that appears to switch the
+    // HP prime into some new protocol mode.
+    // It looks like a SEND_KEY, but the packet ID must be 0xFF I think.
+    prime_raw_hid_pkt raw;
+    memset((void *)&raw, 0, sizeof(raw));
+    raw.size = 8;
+    raw.data[1] = 0xFF;
+    raw.data[2] = 0xEC;
+    // Remaining bytes are 0
```

Comment:
Tbh I'm not sure where exactly the relevant data in that packet ended. The connectivity kit just fills the USB packets up with uninitialized memory past the required section. But these first 8 bytes seemed to be stable in each package at least.